### ISSUE TITLE
There was an error formatting the view

### ISSUE THREAD

**Issue by hakanersu** [adael/SublimePhpCsFixer#3]
Hi, am getting "There was an error formatting the view error."

```python
PHP CS Fixer: Formatting view...
PHP CS Fixer: There was an error formatting the view
PHP CS Fixer: Command: ['/Users/xuma/.composer/vendor/bin/php-cs-fixer', 'fix', '--using-cache=off', '/var/folders/14/_x141cy97l3b731cc1lw30vr0000gn/T/tmpol83v5']
PHP CS Fixer: Error output: b''
PHP CS Fixer: Done. No changes
```
I did check exact command on actual file and its working. It seems there is a problem with reading temp file.

**Comment by t2thec** [comment]
Did you find a solution to this?  I have the same issue.

Cheers

**Comment by adael** [comment]
please give some details, for example OS and sublime text version.

Also, check if the file exists, and try to execute it to view if it's working:

     /Users/xuma/.composer/vendor/bin/php-cs-fixer

**Comment by t2thec** [comment]
Thanks @adael.

for me all seems to be installed OK. If I run:
`php-cs-fixer fix /PATH/Http/Transformers/UserTransformer.php`

All is executed.

On on OSX Sierra 10.12.5
Sublime 3 3126

I'm going to test the path config to see if that has an effect.

Cheers - And thanks for putting this plugin together!

**Comment by hakanersu** [comment]
Hi @adael ,

I did check path. I did test with path config too. i did try with composer and with phar file (/usr/local/bin/php-cs-fixer)

Sublime Text 3 - 3126
OSX 10.12.6

**Comment by waelelsawy** [comment]
I'm also facing this issue, Please fix it asap.

**Comment by t2thec** [comment]
@waelelsawy ‘fix ASAP’? I’m sure @adael would appreciate your input. Feel free to fix yourself it and submit a pull request. 

Don’t forget this is a FREE package that has been build by someone, probably in their spare time. 

Demands like that only make look a little foolish to be honest.

**Comment by erikgall** [comment]
I logged the output to the console and it said `Valet requires Homebrew to be installed on your Mac.`

There was a long discussion here about Valet causing some problems like this before here:

https://laracasts.com/discuss/channels/general-discussion/phpcs-sublime-package-does-not-work-when-laravel-valet-is-installed

I didn't try this but I'm guessing uninstalling valet would resolve the problem..  Or figuring out why Homebrew isn't being found by `laravel/valet` when running the command through Sublime.

**Comment by t2thec** [comment]
Ahhh. Nice one @erikgall 

I have got this working now. I to am using Laravel's Valet.

I had installed php-cs-fixer via composer (globally). I removed this and `brew install php-cs-fixer`.

Then in sublime user `PHP CS Fixer` settings I added the path:
~~~
{
    "path": "/usr/local/bin/php-cs-fixer",
    "config": "/Users/toby/.phpcsfixer",
    "on_save": true,
    "on_load": true
}
~~~

One side note is that if you use a custom config and add the path, `~/.phpcsfixer` didn't work and I had to add the full path. I guess its the path isn't being parsed.

All working for me now. I hope it helps you too.

Cheers
